### ISSUE TITLE
toggle issue in 1.0.1

### ISSUE THREAD

**Issue by willingram** [Lodin/react-vtree#14]
I think there is a problem with `toggle` in 1.0.1.

Having upgraded from 1.0.0, I am receiving the following error on calling toggle for a given node (both in my app and when using the repo's storybook):

```
Uncaught (in promise) TypeError: Cannot read property 'isOpen' of undefined
    at toggle (FixedSizeTree.tsx:76)
```

Thanks for all the work on the project.

**Comment by Lodin** [comment]
Hi @willingram,

Could you provide a bit more info about what the browser and OS you use?

**Comment by ajredovan** [comment]
I believe it is happening in all browsers. It looks like it loses the context of "this" within the toggle function.

**Comment by Lodin** [comment]
@willingram, @ajredovan, thanks for filing the issue. Fixed in [v1.0.2](https://github.com/Lodin/react-vtree/releases/tag/1.0.2).

**Comment by willingram** [comment]
Thanks for the quick fix @Lodin, v1.0.2 working for me now.
